test/setup/main.tf

```diff
@@ -72,6 +72,8 @@ module "gke-project-1" {
   # due to https://github.com/hashicorp/terraform-provider-google/issues/9505 for AP
   default_service_account = "keep"
 
+  deletion_policy   = "DELETE"
```

Comment:
Why is this now needed?

(the lint is also failing on the extra spaces)

Reply:
I can remove it. I just added since we have it in majority of the modules.